app/models/beach_url.rb

```diff
@@ -0,0 +1,95 @@
+ "Jeffreys Bay, South Africa"=>"https://maps.googleapis.com/maps/api/staticmap?zoom=13&size=700x300&maptype=roadmap&markers=color:blue%7Clabel:%7C-34.033333,24.916667",
+ "Unawatuna, Sri Lanka"=>"https://maps.googleapis.com/maps/api/staticmap?zoom=13&size=700x300&maptype=roadmap&markers=color:blue%7Clabel:%7C6.017446899999999,80.2488596",
+ "Cape Maclear, Malawi"=>"https://maps.googleapis.com/maps/api/staticmap?zoom=13&size=700x300&maptype=roadmap&markers=color:blue%7Clabel:%7C-14.016667,34.85",
+ "Las Salinas, Ibiza, Spain"=>"https://maps.googleapis.com/maps/api/staticmap?zoom=13&size=700x300&maptype=roadmap&markers=color:blue%7Clabel:%7C38.8428936,1.3847052",
+ "La Concha, Spain"=>"https://maps.googleapis.com/maps/api/staticmap?zoom=13&size=700x300&maptype=roadmap&markers=color:blue%7Clabel:%7C43.3721287,-3.8526412",
+ "Ffryes Beach, Antigua"=>"https://maps.googleapis.com/maps/api/staticmap?zoom=13&size=700x300&maptype=roadmap&markers=color:blue%7Clabel:%7C17.0746557,-61.8175207",
+ "Puka Beach, Boracay, Philippines"=>"https://maps.googleapis.com/maps/api/staticmap?zoom=13&size=700x300&maptype=roadmap&markers=color:blue%7Clabel:%7C11.9673502,121.924759",
+ "Bandon, Oregon, United States"=>"https://maps.googleapis.com/maps/api/staticmap?zoom=13&size=700x300&maptype=roadmap&markers=color:blue%7Clabel:%7C43.1189978,-124.408448",
+ "An Bang Beach, Hoi An, Vietnam"=>"https://maps.googleapis.com/maps/api/staticmap?zoom=13&size=700x300&maptype=roadmap&markers=color:blue%7Clabel:%7C15.8800584,108.3380469",
+ "Arashi Beach, Aruba"=>"https://maps.googleapis.com/maps/api/staticmap?zoom=13&size=700x300&maptype=roadmap&markers=color:blue%7Clabel:%7C12.610537,-70.0533844",
+ "Byron Bay, Australia"=>"https://maps.googleapis.com/maps/api/staticmap?zoom=13&size=700x300&maptype=roadmap&markers=color:blue%7Clabel:%7C-28.6441616,153.6123788",
+ "Lover’s Beach, Baja California Sur, Mexico"=>"https://maps.googleapis.com/maps/api/staticmap?zoom=13&size=700x300&maptype=roadmap&markers=color:blue%7Clabel:%7C22.8758797,-109.8970271",
+ "Paradise Beach, Rab, Croatia"=>"https://maps.googleapis.com/maps/api/staticmap?zoom=13&size=700x300&maptype=roadmap&markers=color:blue%7Clabel:%7C44.7569096,14.7599675",
+ "Bottom Bay, Barbados"=>"https://maps.googleapis.com/maps/api/staticmap?zoom=13&size=700x300&maptype=roadmap&markers=color:blue%7Clabel:%7C13.1362029,-59.425053",
+ "Meads Bay, Anguilla"=>"https://maps.googleapis.com/maps/api/staticmap?zoom=13&size=700x300&maptype=roadmap&markers=color:blue%7Clabel:%7C18.1819181,-63.13510119999999",
+ "Long Beach, Phu Quoc, Vietnam"=>"https://maps.googleapis.com/maps/api/staticmap?zoom=13&size=700x300&maptype=roadmap&markers=color:blue%7Clabel:%7C10.289879,103.98402",
+ "Hot Water Beach, Coromandel Peninsula, New Zealand"=>"https://maps.googleapis.com/maps/api/staticmap?zoom=13&size=700x300&maptype=roadmap&markers=color:blue%7Clabel:%7C-36.8333333,175.5833333",
```

Comment:
Line is too long. [195/80]<br>Align the elements of a hash literal if they span more than one line.<br>Surrounding space missing for operator '=>'.